db/data.py

```diff
@@ -593,6 +593,107 @@ def load_many_high_level(recordings, map_classes=False):
         return dict(recordings_info)
 
 
+def get_lowlevel_metric_feature(id, path):
```

Comment:
It's possible to remove this function, the associated tests, and the get_data function in the LowLevelMetric class. However, we discussed leaving this functionality incase we use it in the future. We have the git history, and I think it can be removed without too much concern.